packages/flutter/test/physics/newton_test.dart

```diff
@@ -282,8 +277,8 @@ void main() {
     expect(scroll.dx(0.0), closeTo(-7500.0, .0001));
 
     expect(scroll.isDone(0.025), false);
-    expect(scroll.x(0.025), closeTo(320.0, 1.0));
-    expect(scroll.dx(0.25), closeTo(-2982, 1.0));
+    expect(scroll.x(0.025), closeTo(317.0, 1.0));
+    expect(scroll.dx(0.25), closeTo(-4546, 1.0));
 
     expect(scroll.isDone(2.0), true);
     expect(scroll.x(2.0), 0.0);
```

Comment:
the changes in this file seem really sketchy but i assume you've checked that they make sense and aren't regression